docs/develop/api-reference/index.mdx

```diff
@@ -241,7 +264,81 @@ Examples
 | key       | String \| Bytes | The key under which the value's TTL is to be decreased. |
 | ttl       | Duration | Time to live that you want to decrease to. |
 
-### Collection data types (CDTs)
+### ItemGetTtl
+
+For a given key, returns the duration of time remaining (Time To Live) before the item expires from the cache.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| key        | String \| Byte          | Key whose item type should be returned.              |
+
+<details>
+  <summary>Method response object</summary>
+
+  * Cache hit
+  - `remainingTtl()`: Duration
+
+  * Cache miss
+  * Cache error
+
+  See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+## Auth APIs
+
+These APIs are used to manage Momento auth tokens and access.
+
+### GenerateAuthToken
+
+Generates a new Momento Auth token with the specified permissions and expiry.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| scope           | TokenScope      | The permissions to grant to the new token.  Pre-built TokenScope objects are provided by the SDKs. |
```

Comment:
We should list somewhere in the docs what the token scopes are and what permissions they allow